examples/cloudngfw_isolated_design/example.tfvars

```diff
@@ -69,7 +68,6 @@ vpcs = {
       }
     }
     subnets = {
-      # Do not modify value of `set=`, it is an internal identifier referenced by main.tf.
       "10.104.0.0/24"   = { az = "eu-west-1a", set = "app1_vm", nacl = null }

```

Comment:
Are we planning to change `set` in that example too ?

Reply:
You're right I'll modify set for Cloud NGFW as well. Let me change it and push an update

Reply:
As agreed separate PR will be used for the Cloud NGFW example